src/program/render.rs

```diff
@@ -104,12 +105,12 @@ impl<'a> Program<'a> {
     }
 
     fn render_infobar(&mut self) -> Result<(), String> {
-        let text = infobar::Text::from(&self.paths);
+        let text = infobar::Text::update(&self.ui_state.mode, &self.paths);
```

Comment:
@NickHackman 

Here is where the code errors out.

This debug code shows that the `information` and `mode` are empty strings when the program starts to exit.

```rust
    fn render_infobar(&mut self) -> Result<(), String> {
        let text = infobar::Text::update(&self.ui_state.mode, &self.paths);
        dbg!((&text.mode, &text.information));
        // Load the filename texture
        let filename_surface = self
            .screen
            .font
            .render(&text.information)
            .blended(text_color())
            .map_err(|e| e.to_string()).expect("80: error");

```